source/_integrations/tasmota_irhvac.markdown

```diff
@@ -0,0 +1,235 @@
+supported_fan_speeds:
+  description: List of the supported by Your AC fan speeds. Valid values can be obtained by cycling trough all fan speed on Your AC's remote and are then found in `FanSpeed` key in `IRHVAC`. Values have to be double quoted, on new line and prefixed with dash ("-"). Default values might not work for you.
+  required: false
+  type: list
+  default:
+      - "max"
+      - "high"
+      - "medium"
+      - "min"
+supported_swing_list:
+  description: List of the supported by Your AC swing directions. Valid values in the list are `off`, `both`, `vertical` and `horizontal`. If your AC doesn't support horizontal swinging remove -`horizontal` and -`both` from the list
+  required: false
+  type: list
+celsius_mode:
+  description: If the AC temperature is in Celsius of Fahrenheit. Set to `On` for Celsius or `Off` for Fahrenheit. Valid values are `On` and `Off`. Values should be quoted. This option cannot be changed trough HA UI.
+  required: false
+  default: On
+  type: string
+default_quiet_mode:
+  description: If your AC supports quiet mode, and you always want to use it, set it to `On`. This option cannot be changed trough HA UI. Recommended value `Off`.
+  required: false
+  default: Off
+  type: string
+default_turbo_mode:
+  description: If your AC supports turbo mode, and you always want to use it, set it to `On`. This option cannot be changed trough HA UI. Recommended value `Off`.
+  required: false
+  default: Off
+  type: string
+default_econo_mode:
+  description: If your AC supports eco mode, and you always want to use it, set it to `On`. This option cannot be changed trough HA UI. Recommended value `Off`.
+  required: false
+  default: Off
+  type: string
+default_light_mode:
+  description: If your AC supports turning On or Off its own lights, and you always want to use it, set it to `On`. This option cannot be changed trough HA UI. Recommended value `Off`.
+  required: false
+  default: Off
+  type: string
+default_filter_mode:
+  description: If your AC supports filter mode, and you always want to use it, set it to `On`. This option cannot be changed trough HA UI. Recommended value `Off`.
+  required: false
+  default: Off
+  type: string
+default_clean_mode:
+  description: If your AC supports clean mode, and you always want to use it, set it to `On`. This option cannot be changed trough HA UI. Recommended value `Off`.
+  required: false
+  default: Off
+  type: string
+default_beep_mode:
+  description: If your AC supports turning `On` or `Off` its own sounds, and you always want to use it, set it to `On`. This option cannot be changed trough HA UI. Recommended value `Off`.
+  required: false
+  default: Off
+  type: string
+default_sleep_mode:
+  description: If your AC supports sleep mode, and you always want to use it, set it to the value of the `Sleep` key in `HVAC`. This option cannot be changed trough HA UI. Recommended value `-1` or the value of `Sleep` key in `HVAC` when sleep mode is turned ff.
```

Comment:
What is **ff**? (last word in the sentence)

Reply:
It is Off, but without O .. Fixed!